Sources/VLCRendererDiscoverer.m

```diff
@@ -153,16 +154,45 @@ - (void)dealloc
     return [list copy];
 }
 
+- (VLCRendererItem *)discoveredItemsContainsItem:(libvlc_renderer_item_t *)item
+{
+    for (VLCRendererItem *tmpItem in _items) {
+        if (!strcmp(libvlc_renderer_item_name(tmpItem.rendererItem), libvlc_renderer_item_name(item))
+            && !strcmp(libvlc_renderer_item_type(tmpItem.rendererItem), libvlc_renderer_item_type(item))) {
+            return tmpItem;
+        }
+    }
+    return nil;
+}
+
+- (NSArray<VLCRendererItem *> *)renderers
+{
+    return [_items copy];
+}
+
 #pragma mark - Handling libvlc event callbacks
 
-- (void)itemAdded:(VLCRendererItem *)item
+- (void)itemAdded:(NSValue *)item
 {
-    [_delegate rendererDiscovererItemAdded:self item:item];
+    libvlc_renderer_item_t *pointerValue = item.pointerValue;
+    VLCRendererItem *tmp = [self discoveredItemsContainsItem:pointerValue];
+
```

Comment:
Consider renaming both variables to something more self-explaining so the code becomes more readable.